Client/src/Core/MoveAfterRefactor/StoreModules/StepAnalysis/StepAnalysisObservers.ts

```diff
@@ -339,6 +336,38 @@ export const observeStartFormSubmission = (action$: ActionsObservable<Action>, s
   );
 };
 
+export const observeRunAnalysis = (action$: ActionsObservable<Action>, state$: StateObservable<StepAnalysesState>, { wdkService }: EpicDependencies) => {
+  return action$.pipe(
+    filter(isRunAnalysis),
+    withLatestFrom(state$, focusOnPanelById),
+    filter(onTabInSavedState),
+    mergeMap(async ({ panelId, panelState, stepId, strategyId }) => {
+      try {
+        const { status } = await wdkService.runStepAnalysis(stepId, panelState.analysisConfig.analysisId);
+
+        return [
+          finishFormSubmission(panelId, {
+            ...panelState,
+            analysisConfig: {
+              ...panelState.analysisConfig,
+              status
+            },
+            analysisConfigStatus: 'LOADING',
+            formValidationErrors: []
+          }),
+          checkResultStatus(panelId)
+        ];
+      }
+      catch (ex) {
+        const displayName = panelState.analysisConfig.displayName;
+        alert(`Cannot run analysis '${displayName}' at this time. Please try again later, or contact us if the problem persists.`)
+        return EMPTY;
+      }
```

Comment:
I think either this whole catch should be removed, or the alert should be removed and the error rethrown. This will cause the error to get handled by the new error handling mechanism.

Reply:
I'll remove the `catch` - this custom error handling is now obsolete.